manuscript/draft.Rmd

```diff
@@ -99,20 +99,20 @@ of biodiversity [@jetz2007; @thuiller2011], little is known about how
 effectively we can anticipate these dynamics. This is due in part to the long
 time scales over which many ecological forecasts are applied [and the resulting
 difficulty in assessing whether the predicted changes occurred; @dietze2016].
-What we do know comes from a small number of hindcasting studies, where models
-are built using data on species occurrence and richness from the past and
-evaluated on their ability to predict contemporary patterns [e.g., @algar2009;
-@distler2015]. These studies are a valuable first step, but lack several
-components that are important for developing forecasting models with high
-predictive accuracy, and for understanding how well different methods can
-predict the future. These "best practices" for effective forecasting and
-evaluation (Box 1) broadly involve: 1) expanding the use of data to include
-biological and environmental time-series [@tredennick2016]; 2) accounting for
-uncertainty in observations and processes, [@harris2015; @yu2010]; and 3)
-conducting meaningful evaluations of the forecasts by hindcasting, archiving
-short-term forecasts, and comparing forecasts to baselines to determine whether
-the forecasts are more accurate than assuming the system is basically static
-[@perretti2013].
+What we do know comes from a small number of biodiversity studies, where models
```

Comment:
Do we need to drop hindcasting here? I think it's clearer to keep it and based on our definition ("any prediction of observations already available is generally a hindcast") the Blois paper at least still works.

Reply:
Good point, I'll put it back in